### DISCUSSION TITLE
USTX file icons?

### DISCUSSION THREAD

**Discussion by mclt0568** [openutau/OpenUtau#1103] [Ideas]
I always wondered why the USTX file doesn't have its icon and file type association yet.
Here I've made the icon for USTX files,
It has 16x16 48x48 96x96 and 256x256 sizes
[ustx.zip](https://github.com/stakira/OpenUtau/files/14996655/ustx.zip)

Sample screenshots:
![image](https://github.com/stakira/OpenUtau/assets/37445169/3ad4ce39-80e0-42e0-8bd7-b6cc0076699e)
![image](https://github.com/stakira/OpenUtau/assets/37445169/a9f50e1b-9406-44cb-a922-19f6769221c5)
![image](https://github.com/stakira/OpenUtau/assets/37445169/fc90504e-a6ee-4654-9e7c-92fdc07e905c)

**Comment by shuntia** [comment]
You could just use the OU icon...
![image](https://github.com/stakira/OpenUtau/assets/128505810/7041cbda-289f-4164-83a3-32d1f4712080)

**Comment by starbyy** [comment]
I could try doing this! :)
